delete.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Cogctl.Actions.Relays.Delete do
+  use Cogctl.Action, "relays delete"
+
+  def option_spec do
+    []
+  end
+
+  def run(options, args, _config, %{token: nil}=endpoint) do
+    with_authentication(endpoint, &run(options, args, nil, &1))
+  end
+  def run(_options, [], _config, _endpoint), do: display_arguments_error
```

Comment:
The desire is to not require qualified list of arguments for this command, but to allow the user to enter:
```
cogctl delete group1 group2 group3
```

I can include the name in the option_spec and set it to `:undefined`. The function heads for `run` would stay the same and there is no need to use `convert_to_params` since the args has the list of "non-qualified" arguments, but I think it would be documented better.